pkg/trivyoperator/config.go

```diff
@@ -72,6 +72,7 @@ const (
 	KeyMetricsResourceLabelsPrefix         = "metrics.resourceLabelsPrefix"
 	KeyTrivyServerURL                      = "trivy.serverURL"
 	KeyNodeCollectorImageRef               = "node.collector.imageRef"
+	KeyAdditionalReportLables              = ""
```

Comment:
I think we need to have a corresponding field in  the helm config map template and then it should be referenced here.

Reply:
@SamarSidharth defiantly :) the pr its still WIP